engine/apps/slack/scenarios/resolution_note.py

```diff
@@ -655,11 +657,6 @@ def get_invite_bot_tip_blocks(self, channel: str) -> Block.AnyBlocks:
         ]
 
 
-class ReadEditPostmortemStep(ResolutionNoteModalStep):
-    # Left for backward compatibility with slack messages created before postmortems -> resolution note change
-    pass
```

Comment:
_probably_ okay to drop this? I don't see any references to this over the last 6 weeks ([logs](https://ops.grafana-ops.net/goto/I6Rw1l7NR?orgId=1))